genai-perf/genai_perf/inputs/input_constants.py

```diff
@@ -71,7 +71,7 @@ def to_lowercase(self):
 DEFAULT_SYNTHETIC_FILENAME = "synthetic_data.json"
 DEFAULT_WARMUP_REQUEST_COUNT = 0
 DEFAULT_BACKEND = "tensorrtllm"
-PAYLOAD_METADATA_FIELDS = ["timestamp", "delay"]
+PAYLOAD_METADATA_FIELDS = ["timestamp", "delay", "session_id"]
```

Comment:
This part of GAP was designed well, seems like all I needed to do was add this here and the session IDs as supplied via `--input-file=payload:my_payloads.jsonl` would be moved outside of the payload one level up in the input data JSON given to PA.